modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,30 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Advantages and disadvantages of non-versioned and versioned cluster tasks
+
+.Advantages and disadvantages of non-versioned and versioned cluster tasks
+[options="header"]
+|===
+
+| Cluster task variant | Advantages | Disadvantages
+
+| Non-versioned cluster task 
```

Comment:
Because you've given the name of the variant here, you can minimize the sentences on the same row that repeat it.